packages/angular_devkit/build_angular/src/builders/browser/specs/vendor-source-map_spec.ts

```diff
@@ -50,3 +54,90 @@ describe('Browser Builder external source map', () => {
     expect(hasTsSourcePaths).toBe(false, `vendor.js.map not should have '.ts' extentions`);
   });
 });
+
+describe('Identifying third-party code in source maps', () => {
```

Comment:
It's prefer if new tests were created using the new test harness. https://github.com/angular/angular-cli/tree/56320245a126481816a8ccfb0cc248a00be49561/packages/angular_devkit/build_angular/src/builders/browser/tests/behavior

But I guess we can migrate this as a whole eventually.